src/ListPageConfiguration.php

```diff
@@ -108,6 +108,13 @@ class ListPageConfiguration {
    */
   protected $listSource = NULL;
 
+  /**
+   * The languages used in the list.
+   *
+   * @var array
+   */
+  protected $languages = NULL;
```

Comment:
default this to an empty array, not NULL. If we deal with arrays, no need to deal with NULL.